lib/_tools.scss

```diff
@@ -58,33 +58,29 @@
 // We want fine-grain control over which implementations of flexbox we support
 // so we will manually handle prefixes
 //
-// 1. Force a single to fill the avaiable space, required for nested grids
+// 1. Remove any default list styling if the layout is applied to a list
 // 2. Remove any margin and padding which may effect our layout
-// 3. Remove any default list styling if the layout is applied to a list
 //
 // @author Shaun Bent
 //
 @mixin gel-layout() {
+    list-style: none; // [1]
+    direction: flip(ltr, rtl);
+    text-align: flip(left, right);
+
     @if $enhanced {
         display: -webkit-flex;
         display: -ms-flexbox;
         display: flex;
         -webkit-flex-flow: row wrap;
         -ms-flex-flow: row wrap;
         flex-flow: row wrap;
-        -webkit-flex: 1 1 auto; // [1]
```

Comment:
I think this is what is causing the problem. Its not needed in the Codepen but the actual grid seems to break without this.